celery.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM lparkinson/bg_flood:v0.9
+
+WORKDIR /app
+
+# Install Miniconda
+ENV PATH="/root/miniconda3/bin:${PATH}"
+ARG PATH="/root/miniconda3/bin:${PATH}"
+
+RUN apt-get update \
+    && apt-get install -y git \
+                          wget\
+    && rm -rf /var/lib/apt/lists/*
+
+RUN wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+    && mkdir /root/.conda \
+    && bash Miniconda3-latest-Linux-x86_64.sh -b \
+    && rm -f Miniconda3-latest-Linux-x86_64.sh
+RUN conda --version
+
+COPY environment.yml .
+RUN conda env create -f environment.yml
+# Make RUN commands use the new environment:
+SHELL ["conda", "run", "-n", "digitaltwin", "/bin/bash", "-c"]
+
+RUN echo "Check GeoFabrics is installed to test environment"
+RUN python -c "import geofabrics"
+
+COPY selected_polygon.geojson .
+COPY instructions.json .
+COPY src/ src/
+
+EXPOSE 5000
+ENTRYPOINT ["conda", "run", "--no-capture-output", "-n", "digitaltwin", "celery", "-A", "src.tasks", "worker", "-P", "threads", "--loglevel=INFO"]
```

Comment:
@prasannav98 `Dockerfile` has been updated, can you please update this to reflect that?

Reply:
I have found a way using `docker build --target` that may improve this so that we can have just one source of truth. Adding some code to reflect that. It will edit `Dockerfile` and remove `celery.Dockerfile`

Reply:
My code has been added to `docker-compose.yml` and `Dockerfile`, this means you can remove the `celery.Dockerfile`. Test that it works as expected.

Pasting my teams chat message below:
[12:36 pm] Luke Parkinson

> You can user docker compose to build images with my tag name, or if you need to create your own use `docker build --target [celery_worker or backend] -t [yourtagname] .`  or change my tag name in the docker-compose file

> It will take a while to build the first time, because it needs to install libraries etc. so you can use a `docker compose pull` to fetch most of the build layers and then your first build will be faster

Reply:
delete celery.dockerfile